src/api/axios.js

```diff
@@ -0,0 +1,13 @@
+import { createAxiosInstance } from './axiosInstanceFactory';
+import { STORAGE_KEYS, ROUTES } from '../constants/auth.constants';
+
+const getToken = () => localStorage.getItem(STORAGE_KEYS.TOKEN);
+
```

Comment:
Local/session storage should be saved into a service, the service should expose get and set methods, so in the future if we want to add something to it we will do it in one place